www/index.html

```diff
@@ -45,6 +45,27 @@
         <link rel="icon" type="image/png" href="img/icons/kiwix-256.png" sizes="256x256"/>
     </head>
     <body role="application">
+        <!--Bootstrap modal for alert and confirmation pop up-->
+        <div class="modal fade" id="alertModal" tabindex="-1" role="dialog" aria-labelledby="exampleModalLabel" aria-hidden="true">
+            <div class="modal-dialog" role="document">
+                <div class="modal-content">
+                    <div class="modal-header">
+                        <h5 class="modal-title" id="modalLabel">Modal title goes here</h5>
+                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                            <span aria-hidden="true">&times;</span>
+                        </button>
+                    </div>
+                    <div class="modal-body">
+                        <p id="modalText">Modal text goes here.</p>
+                    </div>
+                    <div class="modal-footer" id="modalFooter">
+                        <button type="button" class="btn btn-primary" id="approveModal" data-dismiss="modal">Confirm</button>
+                        <button type="button" class="btn btn-secondary" id="declineModal" data-dismiss="modal">Close</button>
```

Comment:
When it's not used as a confirm dialog box, the user only sees the "decline" button, with label "Cancel", which is confusing.
In this case, the label should be something like "ok" and should have class "btn-primary" (if I understood well). It's probably a third button that needs to be added to this HTML (and handled in the javascript)